api/db-cdr/generate-cdr/make-bq-denormalized-search.sh

```diff
@@ -25,22 +25,47 @@ fi
 # Create bq tables we have json schema for
 schema_path=generate-cdr/bq-schemas
 
+bq --project=$BQ_PROJECT rm -f $BQ_DATASET.cb_search_person
+bq --quiet --project=$BQ_PROJECT mk --schema=$schema_path/cb_search_person.json --time_partitioning_type=DAY --clustering_fields person_id $BQ_DATASET.cb_search_person
+
+bq --project=$BQ_PROJECT rm -f $BQ_DATASET.cb_search_all_events
+bq --quiet --project=$BQ_PROJECT mk --schema=$schema_path/cb_search_all_events.json --time_partitioning_type=DAY --clustering_fields concept_id $BQ_DATASET.cb_search_all_events
+
 ################################################
 # insert person data into cb_search_person
 ################################################
 echo "Inserting person data into cb_search_person"
-bq --quiet --project=$BQ_PROJECT query --nouse_legacy_sql \
-"INSERT INTO \`$BQ_PROJECT.$BQ_DATASET.cb_search_person\`
-    (person_id, gender, race, ethnicity, dob)
-SELECT p.person_id,
+if [ "$BQ_PROJECT:$BQ_DATASET" == "all-of-us-ehr-dev:synthetic_cdr20180606" ]
```

Comment:
Synth dataset doesn't have sex at birth, so we just populate that column with gender to compensate.